### PR TITLE
Fayzulloh lvl1

### PR DESCRIPTION
**Pull request by fayzull0h**
# Purpose
Verify solution to mini c programming challenge.

### CODE REVIEW

File: challenge.c [hunk 1]
```diff
@@ -119,7 +157,13 @@ typedef struct {
...
 
 error_t q10(q10_t *q10) {
-
+    error_t result;
+    if (q9(&(q10->a), &(q10->b)) == 0) {
+        result = SUCCESS;
+    } else {
+        result = FAIL;
+    }
+    return result;
```
Comment (dgobalak) [inline challenge.c]: You _could_ cut this down to a single ternary statement (not required tho)

File: challenge.c [hunk 2]
```diff
@@ -50,10 +59,18 @@ uint16_t q3(uint8_t x, uint8_t y) {
...
+        return -1;
     }
+
+    int32_t sum = 0;
+    for (uint8_t i = 0, overflow = 0; (uint32_t)i + overflow * 255 < arrayLength; i++) {
+        sum += array[i + overflow * 255];
+        if (i == 255) {
+            overflow++;
+        }
+    }
```
Comment (dgobalak) [inline challenge.c]: It'd be more readable to just make `i` have type `uint32_t`

File: challenge.c [hunk 3]
```diff
@@ -2,28 +2,31 @@
 #include <stdbool.h>
 #include <stdlib.h>
 #include <stdio.h>
+#include <inttypes.h>
```
Comment (dgobalak) [inline challenge.c]: Is this header used somewhere in this file? Including `stdint.h` should be sufficient

**Review by dgobalak — CHANGES_REQUESTED** [review]
Looks good. Just a few comments

File: challenge.c (hunk 3 above)
Comment (fayzull0h) [reply challenge.c]: No, it's not. I think I wanted to use a function from there at some point, but I ended up not requiring it and forgot to remove it , sorry.

**Comment by fayzull0h** [comment]
Should I make the changes and resubmit?

**Comment by dgobalak** [comment]
> Should I make the changes and resubmit?

Yep, and then u can get started on the level2 onboarding

**Comment by dgobalak** [comment]
lgtm

**Pull request closed without merging**